src/components/HotkeySettingDialog.vue

```diff
@@ -273,7 +273,7 @@ const duplicatedHotkey = computed(() => {
   );
 });
 
-const deleteHotkey = (action: HotkeyAction) => {
+const deleteHotkey = (action: string) => {
   changeHotkeySettings(action, "");
 };
```

Comment:
コード見てみました！
たぶんこれは`const lastAction = ref("");`側を`HotkeyAction`型にするのが望みかなと思いました。

けどたぶんこれundefined使ったりしないといけなくてちょっと手間だと思うので、一旦fixmeにしておくのはどうでしょう。
```suggestion
// FIXME: actionはHotkeyAction型にすべき
const deleteHotkey = (action: string) => {
  changeHotkeySettings(action, "");
};
```